src/proto/session.rs

```diff
@@ -185,19 +185,57 @@ impl SessionController {
                     }
                 }
 
-                if !self.options.publish {
+                match parameters.style.as_str() {
+                    "PRIMARY" => {}
+
+                    "STREAM" => {}
+
+                    "DATAGRAM" => {
+                        command += format!(
+                            "FROM_PORT={} TO_PORT={}",
```

Comment:
```suggestion
                            "FROM_PORT={} TO_PORT={} ",
```
Whitespace needed at the end so the command is valid